src/saveload/saveload.cpp

```diff
@@ -2647,9 +2647,12 @@ static SaveOrLoadResult DoLoad(LoadFilter *reader, bool load_check)
 
 			DEBUG(sl, 1, "Loading savegame version %d", _sl_version);
 
+			if (_sl_version >= SLV_JGR_EXT_BASE ||
```

Comment:
I think this should use `HasBit()`

Reply:
Fair point. It's a `uint16` so they should be equivalent, but would be clearer.